FeedStoreChallenge/CachedFeedStoreImage.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  CachedFeedStoreImage.swift
+//  FeedStoreChallenge
+//
+//  Created by Hoff Henry Pereira da Silva on 27/07/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(CachedFeedStoreImage)
+final class CachedFeedStoreImage: NSManagedObject {
+	@NSManaged var id: UUID
+	@NSManaged var imageDescription: String?
+	@NSManaged var location: String?
+	@NSManaged var url: URL
+	@NSManaged var cache: Cache
```

Comment:
`cache` is marked as optional in `FeedStore.xcdatamodel`. Please make it optional so that the model and the corresponding class are consistent.

```suggestion
	@NSManaged var cache: Cache?
```